---
layout: docs

title: "google.cloud.forseti.scanner.scanners.config_validator_util.errors module"



hide:
  right_sidebar: true
---
<div class="section" id="module-google.cloud.forseti.scanner.scanners.config_validator_util.errors">
<span id="google-cloud-forseti-scanner-scanners-config-validator-util-errors-module"></span><h1>google.cloud.forseti.scanner.scanners.config_validator_util.errors module<a class="headerlink" href="#module-google.cloud.forseti.scanner.scanners.config_validator_util.errors" title="Permalink to this headline">¶</a></h1>
<p>Config Validator Exceptions.</p>
<dl class="exception">
<dt id="google.cloud.forseti.scanner.scanners.config_validator_util.errors.ConfigValidatorAddDataError">
<em class="property">exception </em><code class="descname">ConfigValidatorAddDataError</code><a class="reference internal" href="_modules/google/cloud/forseti/scanner/scanners/config_validator_util/errors.html#ConfigValidatorAddDataError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#google.cloud.forseti.scanner.scanners.config_validator_util.errors.ConfigValidatorAddDataError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <code class="xref py py-class docutils literal notranslate"><span class="pre">Exception</span></code></p>
<p>ConfigValidator Add Data Error.</p>
</dd></dl>

<dl class="exception">
<dt id="google.cloud.forseti.scanner.scanners.config_validator_util.errors.ConfigValidatorAuditError">
<em class="property">exception </em><code class="descname">ConfigValidatorAuditError</code><a class="reference internal" href="_modules/google/cloud/forseti/scanner/scanners/config_validator_util/errors.html#ConfigValidatorAuditError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#google.cloud.forseti.scanner.scanners.config_validator_util.errors.ConfigValidatorAuditError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <code class="xref py py-class docutils literal notranslate"><span class="pre">Exception</span></code></p>
<p>ConfigValidator Audit Error.</p>
</dd></dl>

<dl class="exception">
<dt id="google.cloud.forseti.scanner.scanners.config_validator_util.errors.ConfigValidatorResetError">
<em class="property">exception </em><code class="descname">ConfigValidatorResetError</code><a class="reference internal" href="_modules/google/cloud/forseti/scanner/scanners/config_validator_util/errors.html#ConfigValidatorResetError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#google.cloud.forseti.scanner.scanners.config_validator_util.errors.ConfigValidatorResetError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <code class="xref py py-class docutils literal notranslate"><span class="pre">Exception</span></code></p>
<p>ConfigValidator Reset Error.</p>
</dd></dl>

<dl class="exception">
<dt id="google.cloud.forseti.scanner.scanners.config_validator_util.errors.ConfigValidatorServerUnavailableError">
<em class="property">exception </em><code class="descname">ConfigValidatorServerUnavailableError</code><a class="reference internal" href="_modules/google/cloud/forseti/scanner/scanners/config_validator_util/errors.html#ConfigValidatorServerUnavailableError"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#google.cloud.forseti.scanner.scanners.config_validator_util.errors.ConfigValidatorServerUnavailableError" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <code class="xref py py-class docutils literal notranslate"><span class="pre">Exception</span></code></p>
<p>ConfigValidator Server Unavailable Error.</p>
</dd></dl>

</div>

